src/Raven.Client/Documents/Session/AbstractDocumentQuery.CounterIncludes.cs

```diff
@@ -25,15 +24,20 @@ public abstract partial class AbstractDocumentQuery<T, TSelf>
                     continue;
                 }
 
-
                 if (kvp.Value.Counters?.Count > 0 == false)
                     continue;
 
-                CounterIncludesTokens.Add(CounterIncludesToken.Create(
-                    kvp.Key,
-                    kvp.Value.Counters.Count == 1
-                        ? AddQueryParameter(kvp.Value.Counters.First())
-                        : AddQueryParameter(kvp.Value.Counters)));
+                if (kvp.Value.Counters.Count == 1)
```

Comment:
We do you have a special  for a single item? Isn't this going to end up with the same thing in the end?

Reply:
done